shell/platform/darwin/macos/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -113,7 +136,7 @@ static BOOL EnableAutocompleteForTextInputConfiguration(NSDictionary* configurat
 
   // Disable if autofill properties indicate a username/password.
   // See: https://github.com/flutter/flutter/issues/119824
-  NSString* contentType = GetAutofillContentType(autofill);
+  NSString* contentType = GetAutofillHint(autofill);
```

Comment:
Consider changing the variable's name as well:

```suggestion
  NSString* hint = GetAutofillHint(autofill);
```

I don't feel strongly about this, feel free to keep as is

Reply:
Good call! Also makes it consistent with the methods above where I use the `hint` terminology.